gatsby-node.js

```diff
@@ -233,6 +272,11 @@ const createTypeDefs = (contentTypesDefinitions) => {
         type ` + capitalize(ctd.name) + ' implements Node {' + tmpDef + `
         }`;
     });
+    Object.keys(additionalDefs).forEach(typeName => {
+        typeDefs = typeDefs + `
+        type ` + typeName + ' {' + additionalDefs[typeName] + `
+        }`;
```

Comment:
Similarly to the above - using template literals for schema building
```suggestion
        typeDefs += `
        type ${typeName} { ${additionalDefs[typeName]}
        }`;
```